sogs.ini.sample

```diff
@@ -1,6 +1,7 @@
 ; SOGS configuration.  Copy this file to sogs.ini and edit as desired.  It can be left mostly (or
 ; entirely) empty: defaults are shown in the comments.
 
+
```

Comment:
stray white space addition that will haunt our souls for several years? :thinking:

Reply:
To be consistent; all the other ini sections have 2 spaces.